sympy/core/tests/test_expr.py

```diff
@@ -1768,4 +1768,5 @@ def test_issue_10755():
 
 def test_issue_11877():
     x = symbols('x')
-    assert (long(integrate(log(0.5-x), (x, 0, 0.5))) == long(-0.846573590279973))
+    y = integrate(x**2/(-x*x), (x, 0, 1))
```

Comment:
@jksuom i changed the test to one you suggest, but this one is a little shorter and easy to understand should we rather use this test?

Not that it really matters but just saying.